src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/details.html.twig

```diff
@@ -117,12 +117,12 @@
   <div class="mt-2" id="order_hook_tabs">
     <ul class="nav nav nav-tabs" role="tablist">
       {# Rendering of hook displayAdminOrderTabLink, we expect tab links #}
-      {{ displayAdminOrderTabLink }}
+      {{ displayAdminOrderTabLink|raw }}
```

Comment:
We propose this fix but it could be better to update renderhook twig function.

Reply:
We got @PierreRambaud approval !
It looks like it can be done by passing `is_safe` option to the `renderHook()` TwigFunction

So we shall pass `is_safe` option to all TwigFunctions in src/PrestaShopBundle/Twig/HookExtension.php I think

Reply:
So actually these functions _already_ have the `is_safe` option but because I put the result in a twig variable, it was escaped again.
So the `|raw` here is good 👍 and no other modification is needed